local/bin/py/build/pull_rbac.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import sys
+import requests
+import json
+from os import getenv
+
+def pull_rbac():
+  api_endpoint = 'https://app.datadoghq.com/api/v2/permissions'
+  headers = {'DD-API-KEY': sys.argv[1], 'DD-APPLICATION-KEY': sys.argv[2]}
+  formatted_permissions_dict = {}
+
+  r = requests.get(api_endpoint, headers=headers)
+
+  if r.status_code == requests.codes.ok:
+    json_result = r.json()
+    data = json_result['data']
+
+    for permission in data:
+      group_name = permission['attributes']['group_name']
+      permission_name = permission['attributes']['name']
+
+      # Remove legacy logs permissions from dictionary before converting to JSON.  These legacy permissions are hard-coded in rbac-permissions-table partial until they can be deprecated.
+      if permission_name == 'logs_live_tail' or permission_name == 'logs_read_index_data':
```

Comment:
you can probably do `if permission_name in ('name1', 'name2')...`

Reply:
Done, thanks!